modules/solid_mechanics/solid_mechanics.mk

```diff
@@ -2,3 +2,6 @@
 ADDITIONAL_NEML2_DIRS += $(APPLICATION_DIR)/test/neml2
 
 include $(APPLICATION_DIR)/contrib/neml2.mk
+
+# Depth of subfolders used to create unity groups
+app_unity_depth = 2
```

Comment:
is there a convention on caps / no caps? This is like a Makefile variable which is intended to go across files

I know caps does not change that

Reply:
I was following the convention for `app_non_unity_dirs`. If there's a different convention, we probably should change both.

Reply:
ok seems fine.